File: src/main/java/com/ject/studytrip/trip/infra/jpa/TripReportStudyLogRepositoryAdapter.java
package com.ject.studytrip.trip.infra.jpa;

import com.ject.studytrip.trip.domain.model.TripReportStudyLog;
import com.ject.studytrip.trip.domain.repository.TripReportStudyLogRepository;
import java.util.List;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Repository;

@Repository
@RequiredArgsConstructor
public class TripReportStudyLogRepositoryAdapter implements TripReportStudyLogRepository {
    private final TripReportStudyLogJpaRepository tripReportStudyLogJpaRepository;

    @Override
    public void saveAll(List<TripReportStudyLog> tripReportStudyLogs) {
        tripReportStudyLogJpaRepository.saveAll(tripReportStudyLogs);
    }
}

File: src/main/kotlin/com/ject/studytrip/trip/infra/jpa/TripReportStudyLogRepositoryAdapter.kt
package com.ject.studytrip.trip.infra.jpa

import com.ject.studytrip.trip.domain.model.TripReportStudyLog
import com.ject.studytrip.trip.domain.repository.TripReportStudyLogRepository
import org.springframework.stereotype.Repository

@Repository
class TripReportStudyLogRepositoryAdapter(
    private val tripReportStudyLogJpaRepository: TripReportStudyLogJpaRepository,
) : TripReportStudyLogRepository {
    override fun saveAll(tripReportStudyLogs: List<TripReportStudyLog>) {
        tripReportStudyLogJpaRepository.saveAll(tripReportStudyLogs)
    }
}
